File: lib/python/player/song.py
from .utility import print_with_buffer


class Song:
    def __init__(self, title, artist, rating):
        self.title = title
        self.artist = artist
        self.rating = rating

    def print(self):
        title = print_with_buffer(self.title, 20)
        artist = print_with_buffer(self.artist, 27)
        rating = str(self.rating)
        print(title + " " + artist + " " + rating)

File: src/lib/dart/lib/player/song.dart
import "package:dart/player/utility.dart" as utility;

class Song {
  String title;
  String artist;
  int rating;

  Song(String title, String artist, int rating) {
    this.title = title;
    this.artist = artist;
    this.rating = rating;
  }

  void printItem() {
    String title = utility.printWithBuffer(this.title, 20);
    String artist = utility.printWithBuffer(this.artist, 27);
    print("${title} ${artist} ${rating}");
  }
}
